aten/src/ATen/core/boxing/boxing.h

```diff
@@ -48,9 +48,11 @@ using supports_boxing =
     not_ok_to_box<guts::decay_t<Args>>...
   >>;
 
+std::string opHandleToName(const OperatorHandle& op);
+
 template<class Result, class... Args>
 Result boxAndCallBoxedFunc(KernelFunction::InternalBoxedKernelFunction* boxed_kernel_func, OperatorKernel* functor, const OperatorHandle& opHandle, Args... args, guts::enable_if_t<!supports_boxing<Result, Args...>::value, int> = 0) {
-  TORCH_INTERNAL_ASSERT(false, "Tried to call KernelFunction::callUnboxed() for a kernel that only has a boxed kernel and doesn't support calling from an unboxed API yet.");
+  TORCH_INTERNAL_ASSERT(false, opHandleToName(opHandle) + " Tried to call KernelFunction::callUnboxed() for a kernel that only has a boxed kernel and doesn't support calling from an unboxed API yet.");
```

Comment:
You don't need to manually `+` the string here: just `opHandleToOpName(opHandle), ": Tried to call"` will be fine (`TORCH_INTERNAL_ASSERT` takes care of the cat'ing). I'm not sure if `name()` returns a `std::string` or not but if it doesn't you can make the function return a `const char*` in this case too.